### PR TITLE
Image feature

### PR DESCRIPTION
**Pull request by Kothalkarpriya**
I have added image features such as Responsive Images and round images.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -468,6 +469,30 @@ margin-bottom: 0;
...
+    display:flex;
+    align-items: center;
+    align-items: center;
+    justify-content: center;
+}
+
+.img-feat{
+    max-width:80%;
+    height:auto;
+}
```
Comment (arp99) [inline style.css:491]: Can please explain what does image-feature and img-feat does? By seeing the classnames it is not clear what styles does they apply

**Review by arp99 — COMMENTED** [review]
Suggested some changes

**Review by JowelTisso — CHANGES_REQUESTED** [review]
You can use px in place where rem values are less than 1rem, Clean the duplicate properties, and the image round property is not working. Please check.

**Comment by ShreyashMahajan** [comment]
code looks good and easy to identify about what thing is being talked about. but one thing I would suggest is that you have used the class name as an image for the container. So you can improve it .

File: index.html [hunk 2]
```diff
@@ -42,7 +43,8 @@ <h4> Avatar is used to represent a user or Profile picture</h4></div>
                             </div>
                         </div>
                         <div class="code">
-                            <iframe src="https://www.thiscodeworks.com/embed/61f93b271e1df40015379416" style="width: 100%; height: 151px;" frameborder="0"></iframe>
+                            <iframe src="https://www.thiscodeworks.com/embed/61f93b271e1df40015379416"
```
Comment (dikshadhote) [inline index.html:46]: Use prettier code formatter or 
do force formatting by using ctrl+shift+p (open command pallete) and select force format to make code for readable

File: index.html [hunk 3]
```diff
@@ -466,9 +474,33 @@ <h2>Sunset In Aurangabad</h2>
...
+                <article class="container-info">
+                    <div class="info">
+                        <h4>Image is important to be responsive with which can be accessed easily</h4>
+                    </div>
+                    <div class="image-container">
+                        <div class="text-badge">Responsive Image</div>
+                        <div class="image-feature">
+                            <img src="image.jpg" alt="sunset" class="img-feat">
+                        </div>
+                        <div class="code">
```
Comment (dikshadhote) [inline index.html]: You can use better class name than code...For Eg: code-container

File: style.css [hunk 4]
```diff
@@ -468,6 +469,30 @@ margin-bottom: 0;
...
+
+.img-round{
+    border-radius:50%;
+}
 /* ----------------------------------------------------------- */
 .code{
```
Comment (dikshadhote) [inline style.css:497]: use separate classes for flex. justify-content and alignment

**Comment by t007rushi** [comment]
In a preview of Cards You can see Options footer are not aligned in-line (you can refer your horizontal card) they are stacked increase width of the card this is major change needed

File: index.html (hunk 2 above)
Comment (Kothalkarpriya) [reply index.html:46]: I used a prettier code formatter now. Thanks

File: index.html (hunk 3 above)
Comment (Kothalkarpriya) [reply index.html]: Converted class name code to code-container

File: style.css (hunk 4 above)
Comment (Kothalkarpriya) [reply style.css:497]: Ok I will use it

File: style.css (hunk 1 above)
Comment (Kothalkarpriya) [reply style.css:491]: .image-feature is used to style the block of image and style it as making its alignment to the center. img-feat used for sizing the image

**Comment by Kothalkarpriya** [comment]
> You can use px in place where rem values are less than 1rem, Clean the duplicate properties, and the image round property is not working. Please check.

I used border-radius to be 50% but the image height is more to be in a circle that's why its shape is different

**Comment by Kothalkarpriya** [comment]
> code looks good and easy to identify about what thing is being talked about. but one thing I would suggest is that you have used the class name as an image for the container. So you can improve it.

Okay, I will look into it

File: index.html (hunk 3 above)
Comment (Kothalkarpriya) [reply index.html]: changed the class name, thanks

File: index.html (hunk 2 above)
Comment (Kothalkarpriya) [reply index.html:46]: I used code formatter, Thanks

File: style.css (hunk 1 above)
Comment (Kothalkarpriya) [reply style.css:491]: .image-feature is used to style the block of image and style it as making its alignment to the center. img-feat used for sizing the image can I resolve the issue now?

Comment (arp99) [reply style.css:491]: ok

**Review by arp99 — APPROVED** [review]
Looks good

**Comment by Kothalkarpriya** [comment]
> Looks good

Thank you

**Pull request merged**